README.md

```diff
@@ -27,11 +27,15 @@ This build plugin is a utility for supporting Next.js on Netlify. To enable serv
 
 There are two ways to install the plugin in your Next.js on Netlify site.
 
-First:
+**First:**
```

Comment:
With these "first" and "second" headings, I wonder if people who skip the intro sentence might read them as two steps. Two possible ways to avoid this:
- Option 1 and Option 2
- UI install and File-based install

Reply:
oooo so true great callout!!!!!!!!!!!!